core/dbt/config/runtime.py

```diff
@@ -314,10 +314,11 @@ def warn_for_unused_resource_config_paths(
             return
 
         msg = UNUSED_RESOURCE_CONFIGURATION_PATH_MESSAGE.format(
-            len(unused),
-            '\n'.join('- {}'.format('.'.join(u)) for u in unused)
+            printer.red(len(unused)),
+            '\n'.join('\t\t- {}'.format('.'.join(u)) for u in unused)
```

Comment:
I really think two tabs is a lot. Do you think six or eight spaces would be reasonable here? My terminal shows this as a 16-space indent!

Reply:
I took out the formatting (see above comment). Thanks!